pom.xml

```diff
@@ -328,6 +328,26 @@
                             <goal>test-jar</goal>
                         </goals>
                     </execution>
+                    <execution>
+                        <id>jar-jdk8</id>
+                        <goals>
+                            <goal>jar</goal>
+                        </goals>
+                        <configuration>
+                            <classifier>jdk8</classifier>
+                            <classesDirectory>${project.build.outputDirectory}_jdk8</classesDirectory>
```

Comment:
Similar to the `maven-compiler-plugin` above. The default `default-jar` execution creates a java 8 jar, therefore the `jar-jdk8` execution creates a duplicate jar with the same size and near identical content.

The following jars are built:
- `302843 10 Mar 08:49 core-2.0.1-SNAPSHOT.jar`
- `302843 10 Mar 08:49 core-2.0.1-SNAPSHOT-jdk8.jar`
- `303078 10 Mar 08:49 core-2.0.1-SNAPSHOT-jdk11.jar`
- `326327 10 Mar 08:49 core-2.0.1-SNAPSHOT-tests.jar`

Reply:
Maybe a silly question but do we know if these will all get released? I presume that is the desired outcome that we release java 8 and 11 versions, but just checking we don't need to change the release profile?

Reply:
We shouldn't need to change the release profile (`ossrh-release`). This doesn't specify any particular jars, so it should just upload them all.